demos/docker_images.rst

```diff
@@ -1,6 +1,6 @@
-================
-Using OpenAMP CI
-================
+==========================
+OpenAMP Demo Docker images
+==========================
 
 The OpenAMP CI solution is just in it's infancy at the moment and this doc is a work-in-progress. Please keep it up to date and don't keep historical information (that is what wiki history is for).
```

Comment:
does make sense to speak about "OpenAMP CI solution"?

Reply:
@wmamills @edmooring ?

Reply:
@wmamills Has taken on the action to scrub this content.  This commit is just to address his comment to move and rename:

     Using OpenAMP CI
	move to Samples & Demos under name of
	"OpenAMP Demo Docker images"
	I will be scrub owner for this page